src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IAnonymousFunctionOperation.vb

```diff
@@ -0,0 +1,501 @@
+                  Instance Receiver: 
+                    IFlowCaptureReferenceOperation: 0 (OperationKind.FlowCaptureReference, Type: C, IsImplicit) (Syntax: 'New C With  ... nction() 1}')
+              Right: 
+                IDelegateCreationOperation (OperationKind.DelegateCreation, Type: System.Func(Of System.Int32), IsImplicit) (Syntax: 'Function() 1')
+                  Target: 
+                    IFlowAnonymousFunctionOperation (Symbol: Function () As System.Int32) (OperationKind.FlowAnonymousFunction, Type: null) (Syntax: 'Function() 1')
+
+            ISimpleAssignmentOperation (OperationKind.SimpleAssignment, Type: C, IsImplicit) (Syntax: 'c1, c2 As N ... nction() 1}')
+              Left: 
+                ILocalReferenceOperation: c1 (IsDeclaration: True) (OperationKind.LocalReference, Type: C, IsImplicit) (Syntax: 'c1')
+              Right: 
+                IFlowCaptureReferenceOperation: 0 (OperationKind.FlowCaptureReference, Type: C, IsImplicit) (Syntax: 'New C With  ... nction() 1}')
+
+            IFlowCaptureOperation: 1 (OperationKind.FlowCapture, Type: null, IsImplicit) (Syntax: 'New C With  ... nction() 1}')
+              Value: 
+                IObjectCreationOperation (Constructor: Sub C..ctor()) (OperationKind.ObjectCreation, Type: C) (Syntax: 'New C With  ... nction() 1}')
+                  Arguments(0)
+                  Initializer: 
+                    null
+
+            ISimpleAssignmentOperation (OperationKind.SimpleAssignment, Type: System.Void) (Syntax: '.A = Function() 1')
+              Left: 
+                IPropertyReferenceOperation: Property C.A As System.Func(Of System.Int32) (OperationKind.PropertyReference, Type: System.Func(Of System.Int32)) (Syntax: 'A')
+                  Instance Receiver: 
+                    IFlowCaptureReferenceOperation: 1 (OperationKind.FlowCaptureReference, Type: C, IsImplicit) (Syntax: 'New C With  ... nction() 1}')
+              Right: 
+                IDelegateCreationOperation (OperationKind.DelegateCreation, Type: System.Func(Of System.Int32), IsImplicit) (Syntax: 'Function() 1')
+                  Target: 
+                    IFlowAnonymousFunctionOperation (Symbol: Function () As System.Int32) (OperationKind.FlowAnonymousFunction, Type: null) (Syntax: 'Function() 1')
+
+            ISimpleAssignmentOperation (OperationKind.SimpleAssignment, Type: C, IsImplicit) (Syntax: 'c1, c2 As N ... nction() 1}')
+              Left: 
+                ILocalReferenceOperation: c2 (IsDeclaration: True) (OperationKind.LocalReference, Type: C, IsImplicit) (Syntax: 'c2')
+              Right: 
+                IFlowCaptureReferenceOperation: 1 (OperationKind.FlowCaptureReference, Type: C, IsImplicit) (Syntax: 'New C With  ... nction() 1}')
+
+        Next (Regular) Block[B2]
+            Leaving: {R1}
+    
+    {   Function () As System.Int32
```

Comment:
It was less obvious with local functions (that have unique names), but perhaps we should add some kind of identifier to the dumper for lambdas, as it would be pretty hard to tell which one is which if there were multiple lambdas in this region.

Reply:
>  it would be pretty hard to tell which one is which if there were multiple lambdas in this region.

That is true. And it is even worse for C#, which is captured in a prototype comment in the verifier. Given the time constraints, I am not planning to address this issue before we ship.

---
In reply to: [192223413](https://github.com/dotnet/roslyn/pull/27282#discussion_r192223413) [](ancestors = 192223413)

Reply:
Ok, I'll link from [https://github.com/dotnet/roslyn/issues/22559](https://github.com/dotnet/roslyn/issues/22559).

---
In reply to: [192224403](https://github.com/dotnet/roslyn/pull/27282#discussion_r192224403) [](ancestors = 192224403,192223413)